tests/stdlib/tmersenne.nim

```diff
@@ -6,6 +6,13 @@ template main() =
   doAssert mt.getNum == 407788156'u32
   doAssert mt.getNum == 1071751096'u32
   doAssert mt.getNum == 3805347140'u32
+  doAssert sample(mt, @[30, 20, 10]) == 20
+  doAssert sample(mt, @[30.4, 20.54, 10.5]) == 10.5
+  doAssert sample(mt, @['a', 'b', 'c']) == 'b'
+  doAssert sample(mt, @["abc", "def", "ghi"]) == "def"
+  
+
+
```

Comment:
These tests won't compile without the proc.

Reply:
Sorry, stupid of me. Fixing.